### ISSUE TITLE
Docker container - error in Alpine 3.7

### ISSUE THREAD

**Issue by lhilton** [dweidenfeld/plexdrive#285]
I am setting up a docker with plexdrive in it using Alpine 3.7 as the base. Attempting to run the plexdrive binary gives me this error:

```No such file or directory```

I assume I am missing dependencies in the container, as I see this working in debian based containers. Is there a list of system dependencies I need to worry about somewhere?

**Comment by zenjabba** [comment]
Docker and fuse don’t work great together inside the docker. If you get it working, that would be amazing...

**Comment by lhilton** [comment]
Docker and fuse work just fine inside a container. There are tonnes of examples of this, and I am using it in many other places (including with plexdrive).

**Comment by zenjabba** [comment]
So you can expose the fuse mount OUTSIDE the docker?

**Comment by barlock** [comment]
Yeah here's an example I've been using as a base for some of my images https://github.com/madslundt/docker-cloud-media-scripts

**Comment by Dulanic** [comment]
@barlock Having used that specific docker, just a FYI, if the mount ever has to remount etc.... it stops working. That is the main issue everyone will have with FUSE remounts and docker. Unless, there is a work around but this is what I kept seeing...

Madslunt himself even said there are some issues.

> Yes some people have experience the issue you address here. I'm looking into it but so far it looks like docker isn't happy about mounting folders and then sharing them with other containers.

https://github.com/madslundt/docker-cloud-media-scripts/issues/16

```
root@mediaserver:/local-decrypt# cd ..
root@mediaserver:/# cd local-media
bash: cd: local-media: Transport endpoint is not connected
root@mediaserver:/# cd cloud-decrypt
bash: cd: cloud-decrypt: Transport endpoint is not connected
```

**Comment by lhilton** [comment]
I didn't know that. I run these bits inside of a couple of virtual machines with docker on them. When my test script sees a failure (like not being able to read the cloud-decrypt folder) it just reboots the entire host itself. There are some other issues in my stack so I stopped auto-restarting the container and just started rebooting the virtual host.

**Comment by Shadoukun** [comment]
Like @barlock I'm also attempting to build an alpine docker image using https://github.com/madslundt/docker-cloud-media-scripts
as an example.

The ubuntu version runs fine, but the alpine version throws `No such file or directory`, even though `docker ls` shows that plexdrive is clearly there and executable.

So this isn't a Docker fuse issue, its an alpine plexdrive issue it seems.

**Comment by felixbuenemann** [comment]
I'd check `ldd /path/to/pexdrive` you probably have incompatible or missing libraries and need to recompile plexdrive for musl libc used by alpine.

**Comment by Shadoukun** [comment]
That was the issue.

Thanks, worked fine after rebuilding it with musl.
